CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ set(LIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libs)
 # Downloads CPM if not already downloaded. CPM is an easy-to-use package manager nicely integrated with CMake.
 include(cmake/cpm.cmake)
 
+set(JUCE_COPY_PLUGIN_AFTER_BUILD TRUE)
```

Comment:
I don't want to have this on because it may cause errors on Windows (no permission).

Reply:
removed it